.github/workflows/test.yml

```
@@ -9,11 +9,23 @@ jobs:
   tests:
     runs-on: ${{ matrix.platform }}
     strategy:
-      max-parallel: 5
+      max-parallel: 7
       matrix:
-        platform: [ubuntu-latest]
-        tox-env: ["py39", "py310", "py311", "py312", "py313"]
-
+        include:
+          - platform: ubuntu-latest
+            tox-env: "py39"
+          - platform: ubuntu-latest
+            tox-env: "py310"
+          - platform: ubuntu-latest
+            tox-env: "py311"
+          - platform: ubuntu-latest
+            tox-env: "py312"
+          - platform: ubuntu-latest
+            tox-env: "py313"
+          - platform: macos-13
+            tox-env: "py313"
+          - platform: macos-latest
+            tox-env: "py313"
```

[nits] Could you add a comment like below?
```suggestion
          # test only on latest python for macos
          - platform: macos-13
            tox-env: "py313"
          - platform: macos-latest
            tox-env: "py313"
```

---

OK! I have inserted the comment!